.github/workflows/ci_tests.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Set up Python 3.9
       uses: actions/setup-python@v4
       with:
-        python-version: 3.9
```

Comment:
Why the change from no quotes to with quotes?

Reply:
Because this is the correct way to assign this value. One will find that this is true when trying to use `3.10`. This will fail unless wrapped in quotes.